specification/dartLangSpec.tex

```diff
@@ -14866,19 +14947,12 @@ \subsubsection{Asynchronous For-in}
 the subscription $u$ is canceled by evaluating \code{\AWAIT{} v.cancel()}
 where $v$ is a fresh variable referencing the stream subscription $u$.
 If that evaluation throws,
-execution of $f$ throws the same exception object and stack trace.
-Otherwise execution of $f$ throws with $e$ as exception object and $st$ as stack trace.
-
-\LMHash{}%
-When $u$ is done, execution of $f$ completes normally.
+execution of $s_b$ throws the same exception object and stack trace.
+Otherwise execution of $s_b$ throws
+with $e$ as exception object and $st$ as stack trace.
 
```

Comment:
I can't find the error-event paragraph. Is it just hidden, or should it be here:

> ```latex
> \LMHash{}%
> If $u$ emits an error event with error object \metavar{error} and associated stack trace \metavar{stack},
> then $u$ is cancelled and completed in the same way as if $s_b$ had completed by throwing 
> \metavar{error} and \metavar{stack}.

Reply:
Added after `$st$ as stack track` (not after `completes normally`).